New entry: improving the email sent to candidates about new interviews (#663)
DFE-Digital/bat-design-history: app/posts/apply-for-teacher-training/2022-07-13-improving-the-email-sent-to-candidates-about-new-interviews.md

---
title: Improving the email sent to candidates about new interviews
description: We now include the course subject and code. We also show details separately rather than in a single section.

> date: 2022-07-13
> ---

Reviewer: We could link to the pull request as a related item maybe?

Also updated date to today.

```suggestion
date: 2022-08-12
related:
  items:
    - text: Pull request implementing this change
      href: https://github.com/DFE-Digital/apply-for-teacher-training/pull/7276
---
```

{% from "email/macro.njk" import appEmail %}

> The email sent to candidates when an interview is set up does not currently include the course name or code.

Reviewer: Suggest changing this to the past tense for consistency with the rest of the post.

```suggestion
The email sent to candidates when an interview is set up did not include the course name or code.
```

We wanted to add these to the email and also reflect [changes we made to the email sent when an interview is updated](/manage-teacher-training-applications/emailing-candidates-about-upcoming-interviews-when-their-course-is-changed/).

While making these changes we looked at examples of text entered for:

- address or online meeting details
- additional details

> We found that providers often give large amounts of information, sometimes including multiple paragraphs. This does not display well in the existing design so we decided to change it.

Reviewer: Change to past tense?

```suggestion
We found that providers often give large amounts of information, sometimes including multiple paragraphs. This did not display well in the design so we decided to change it.
```

Could we elaborate on "not display well"?
